src/main/java/th/project/enterprise/Service/UserService.java

```diff
@@ -28,28 +26,53 @@ public void creatUser(Customer user) {
         user.setPassword(encoder.encode(user.getPassword()));
         userRepoistory.save(user);
     }
+    public void creatUser(User user) {
+        BCryptPasswordEncoder encoder = new BCryptPasswordEncoder();
+        user.setPassword(encoder.encode(user.getPassword()));
+        userRepoistory.save(user);
+    }
 
     public Customer findByEmail(String email) {
         return userRepoistory.getUserByEmail(email);
     }
+    public User findUserByEmail(String email) {
+        return userRepoistory.getUserByEmail(email);
+    }
 
-    public boolean isUserPresent(String email) {
+    public boolean isCustomerPresent(String email) {
```

Comment:
i am little bit cofused, because we have now a methode called isCustomerPresent() that create Customer Object and it's look like, the method is not used, and we still have isUserPresent, that create User Object.

do we need IsEmployeePresent ??